src/bidiServerRunner.ts

```diff
@@ -27,19 +27,38 @@ export class BidiServerRunner {
   // Once run, waits for connection. For each connection, calls `onOpen` and stores the returned `stateObject`.
   // `stateObject` passed to `onClose` afterwards.
   static run(
+    bidiPort: number,
     onNewBidiConnectionOpen: (bidiServer: IServer) => Promise<BrowserProcess>,
     onBidiConnectionClosed: (browserProcess: BrowserProcess) => void
   ) {
     const self = this;
-    const bidiPort = parseInt(process.env.PORT) || 8080;
 
     const server = http.createServer(function (request, response) {
       debugInternal(new Date() + ' Received request for ' + request.url);
-      response.writeHead(404);
+
+      // Needed for WPT compatibility.
```

Comment:
This was already landed in https://github.com/GoogleChromeLabs/chromium-bidi/pull/7/files, why is it also showing up in this PR despite https://github.com/GoogleChromeLabs/chromium-bidi/pull/7 being merged?

Hmm, I think it's because there are conflicts in `package.json`, so GitHub can't merge into `main` in order to compare against `main`...

Reply:
done